playground/config/transformer-now.js

```diff
@@ -0,0 +1,44 @@
+const fs = require('fs');
+const path = require('path');
+
+const regions = {
+  eu: ['bru'],
+  us: ['sfo'],
+};
+const headersStaticFiles = { 'cache-control': 's-maxage=31536000,immutable' };
+
+// This transformer will generate a `now.json` config file, based on the application
+// environment config and custom headers.
+module.exports = ({ env, headers }) => {
+  const environmentKey = `${env.env}-${env.location}`;
+  const config = {
+    version: 2,
+    public: true,
+    name: `mc-app-state-machines-${environmentKey}`,
+    alias: `mc-app-state-machines-${environmentKey}`,
+    regions: regions[env.location],
+    builds: [
+      { src: 'public/**', use: '@now/static' },
+      { src: 'config/fallback-route.js', use: '@now/node' },
+    ],
+    routes: [
+      {
+        src: '/(.*).(js.map|js|css|txt|html|png)',
```

Comment:
should this be configurable? what if I want to host some svg? 🤔

Reply:
or PDFs 😄

Reply:
It’s up to you to configure it the way you like it, this is just an example for the playground app.
We don’t ship a transformer out of the box.